addons/base_import/models/base_import.py

```diff
@@ -719,6 +719,17 @@ def _parse_float_from_data(self, data, index, name, options):
             if not line[index]:
                 continue
             thousand_separator, decimal_separator = self._infer_separators(line[index], options)
+
+            tmp_value = line[index]
+            if 'e' in line[index] or 'E' in line[index]:
```

Comment:
```suggestion
            if 'E' in line[index] or 'e' in line[index]:
```
'E' is the most used notation (Excel)